File: src/cn/langya/sun/utils/ClientUtils.java
package cn.langya.sun.utils;

import net.minecraft.util.text.ITextComponent;
import net.minecraft.util.text.TextComponentString;
import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

public class ClientUtils extends Utils{
    public static final Logger logger = LogManager.getLogger("SunClient");

    public static void loginfo(String string) {
        logger.info(string);
    }

    public static void chatlog(String message) {
        final String text = String.valueOf(message);
        mc.ingameGUI.getChatGUI().printChatMessage(new TextComponentString(text));
    }

}

File: src/cn/langya/sun/utils/ClientUtils.kt
package cn.langya.sun.utils

import org.apache.logging.log4j.LogManager
import org.apache.logging.log4j.Logger

/**
 * @author LangYa
 * @ClassName ClientUtils
 * @date 2023/12/14 21:07
 * @Version 1.0
 */

object ClientUtils {
    val logger: Logger = LogManager.getLogger("SunClient")

    fun loginfo(string: String) {
        logger.info(string)
    }

}